core/db_models/fields/EE_Foreign_Key_String_Field.php

```diff
@@ -14,6 +36,7 @@ public function prepare_for_set($value_inputted_for_field_on_model_object)
         if ($this->is_model_obj_of_type_pointed_to($value_inputted_for_field_on_model_object)) {
             $value_inputted_for_field_on_model_object = $value_inputted_for_field_on_model_object->ID();
         }
-        return strtoupper(wp_strip_all_tags($value_inputted_for_field_on_model_object));
+        $clean_value = wp_strip_all_tags($value_inputted_for_field_on_model_object);
+        return $this->is_uppercase ? strtoupper($clean_value) : $clean_value;
```

Comment:
The reason for this change was that the `FIN_belongsTo` for `Form_Element` was converted to UPPERCASE, this making the relation invalid for queries.

Reply:
oh yuk... i'm assuming Michael did that because the country table is the main table that uses string IDs, which are the ISO codes like `USA`, `CAD`, `IND`, etc and those are all capitalized. 

That's business logic that should be pushed into the Country model itself and not be part of EEM_Base